simulator/scheduler/config/plugin.go

```diff
@@ -7,15 +7,28 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework/runtime"
 )
 
```

Comment:
The change in this file is mostly the refactoring.
We don't need to handle each extension point; only multi point should be enough